.valgrind.suppressions

```diff
@@ -179,3 +179,18 @@
    fun:notification_load_icon_with_scaling
    ...
 }
+
+{
+   # Something new on Ubuntu Noble Numbat
```

Comment:
Just to know, how did you find this out? Is this a problem with the new versions of rsvg?

Reply:
I'm not sure what the reason is. I didn't encounter the same issue on Arch Linux which uses the same version of the lib.

I will take another look what the data looks like on our side in that code path.

Reply:
<details>
  <summary>Full trace</summary>

```
==4793==    at 0x6106346: glib::param_spec::ParamSpecStringBuilder::build (in /usr/lib/x86_64-linux-gnu/librsvg-2.so.2.50.0)
==4793==    by 0x5DC5DC9: std::sys_common::once::futex::Once::call (in /usr/lib/x86_64-linux-gnu/librsvg-2.so.2.50.0)
==4793==    by 0x5DC53AF: std::sync::once_lock::OnceLock<T>::initialize (in /usr/lib/x86_64-linux-gnu/librsvg-2.so.2.50.0)
==4793==    by 0x5DEBA50: glib::subclass::object::<impl glib::subclass::types::IsSubclassable<T> for glib::object::Object>::class_init (in /usr/lib/x86_64-linux-gnu/librsvg-2.so.2.50.0)
==4793==    by 0x5DF0839: glib::subclass::types::class_init (in /usr/lib/x86_64-linux-gnu/librsvg-2.so.2.50.0)
==4793==    by 0x4C0DA7C: UnknownInlinedFun (gtype.c:2319)
==4793==    by 0x4C0DA7C: g_type_class_ref (gtype.c:3034)
==4793==    by 0x4BF5161: g_object_new_with_properties (gobject.c:2745)
==4793==    by 0x61082E8: glib::object::Object::new_internal (in /usr/lib/x86_64-linux-gnu/librsvg-2.so.2.50.0)
==4793==    by 0x5DE299C: rsvg_handle_new (in /usr/lib/x86_64-linux-gnu/librsvg-2.so.2.50.0)
==4793==    by 0x485B44B: gdk_pixbuf__svg_image_load_increment (io-svg.c:118)
==4793==    by 0x4B2C4F0: generic_load_incrementally (gdk-pixbuf-io.c:1043)
==4793==    by 0x4B31CE6: gdk_pixbuf_new_from_file (gdk-pixbuf-io.c:1148)
==4793==    by 0x12F7EC: notification_setup_raw_image (helpers.c:9)
==4793==    by 0x137926: notification_load_icon_with_scaling (notification.c:142)
==4793==    by 0x138A72: test_notification_icon_scaling_toosmall (notification.c:168)
==4793==    by 0x138A72: suite_notification (notification.c:260)
==4793==    by 0x146B2B: greatest_run_suite (test.c:33)
==4793==    by 0x146B2B: greatest_run_suite (test.c:33)
==4793==    by 0x125535: main (test.c:57)
```
</details>

I couldn't find anything wrong from our side: `notification_setup_raw_image` just calls `gdk_pixbuf_new_from_file` with a valid path (and a seemingly valid SVG file) and `NULL` for the error. So, I'd say we stick with the suppression for the time being.